runtime/src/main.rs

```diff
@@ -36,14 +36,15 @@ const BANNER: &str = r#"
 pub extern "C" fn entry_point() -> ! {
     cprintln!("{}", BANNER);
     if let Some(mut fht) = caliptra_common::FirmwareHandoffTable::try_load() {
-        let mut drivers = unsafe { Drivers::new_from_registers(&mut fht) };
-
-        // Indicator to SOC that RT firmware is ready
-        drivers
-            .soc_ifc
-            .regs_mut()
-            .cptra_flow_status()
-            .write(|w| w.ready_for_runtime(true));
+        let mut drivers = match unsafe { Drivers::new_from_registers(&mut fht) } {
```

Comment:
Looks like a merge conflict. Did you intent to remove the ready_for_runtime on the left? I don't see it re-added elsewhere.

Reply:
I was thinking this was a duplicate of https://github.com/chipsalliance/caliptra-sw/blob/012e20c20521ed3a56956a5b0249384365b23f67/runtime/src/lib.rs#L201C59-L201C59

I didn't realize boot_status and flow_status are different. I'll add this back.

Reply:
moved to `handle_mailbox_commands` alongside the boot status reporting.